lib/graphql/schema.rb

```diff
@@ -773,9 +773,9 @@ def max_depth(new_max_depth = nil)
         end
       end
 
-      def orphan_types(new_orphan_types = nil)
-        if new_orphan_types
-          @orphan_types = new_orphan_types
+      def orphan_types(*new_orphan_types)
+        if new_orphan_types.any?
+          @orphan_types = new_orphan_types.flatten
```

Comment:
Hmm, can you help me understand how this change fits into the feature?

Reply:
Currently `GraphQL::Schema` requires an array and `GraphQL::Schema::Union` accepts varargs:

```ruby
class MySchema < GraphQL::Schema
  orphan_types [Types::CircleType, Types::SquareType]
end

class Types::ShapeUnion < GraphQL::Schema::Union
  possible_types Types::CircleType, Types::SquareType
end
```
This change was done to make `GraphQL::Schema.orphan_types`, `GraphQL::Schema::Interface.orphan_types` and `GraphQL::Schema::Union.possible_types` all accept varargs to make them consistent. I left the `GraphQL::Schema.orphan_types` support for an array argument for backwards compatibility but maybe for consistency we should make all of these APIs accept varargs or an array?